### PR TITLE
Laravel 9 Support

### PR DESCRIPTION
**Pull request by ibnu-ja**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -12,11 +12,13 @@ jobs:
       fail-fast: true
       matrix:
         php: [8.0]
```
Comment (DarkGhostHunter) [inline .github/workflows/php.yml:14]: Needs to add PHP 8.1 support too on the workflow.

**Review by DarkGhostHunter — CHANGES_REQUESTED** [review]
PHP 8.1 should be added to the workflow.

File: .github/workflows/php.yml (hunk 1 above)
Comment (ibnu-ja) [reply .github/workflows/php.yml]: `PHP Fatal error:  During inheritance of IteratorAggregate: Uncaught ErrorException: Return type of Symfony\Component\HttpFoundation\ParameterBag::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/runner/work/Laraconfig/Laraconfig/vendor/symfony/http-foundation/ParameterBag.php:205`

looks like it also need newer of symfony
How about bump min version to [v8.74.0](https://github.com/laravel/framework/releases/tag/v8.74.0)?

**Comment by DarkGhostHunter** [comment]
Go ahead then. 

Italo Baeza C.

**Pull request closed without merging**